### PR TITLE
Form section

### PR DESCRIPTION
**Pull request by PrinceBadru**
## Mobile Finish.
 -Add form tags.
 -Add form text areas.
 -Add form place holders.
 -Add Email field that accepts only emails.
 -add Message field that accepts up to 500 chars.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,154 +36,250 @@ <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer
...
+            <ul class ="socials">
+                <li><a href="https://twitter.com/BadruNdawula"><img src="./images/twitter.jpg" alt=""></a></li>
+                <li><a href="https://www.linkedin.com/in/badru-ndawula-32b371236/"><img src="./images/Linkedin icon.jpg" alt=""></a></li>
+                <li><a href="https://github.com/PrinceBadru"><img src="./images/github.jpg" alt=""></a></li>
+                <li><a href="#"><img src="./images/medium.jpg" alt=""><a></li>
+                <li><a href="#"><img src="./images/angellist.jpg" alt=""></a></li>
+            </ul> 
+            </footer>
+            
+>>>>>>> Stashed changes
```
Comment (OyePriscilla) [inline index.html:282]: - [ ] Kindly remove this comments for cleaner codebase. 👍 Please do the same for line 234 and  and 270 as well. 😄

**Review by OyePriscilla — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -37,154 +36,250 @@ <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer
...
+                       <li>
+                           <label for="msg">
+                               <textarea name="user_message" id="msg" cols="30" rows="5" maxlength="500" placeholder="Enter text here" required></textarea>
+                           </label>
+                       </li>
+                       <li class="button back-image">
+                           <button type="submit" class="btn">Get in touch</button>
+                       </li>
+                   </ul>
+               </form>
```
Comment (OyePriscilla) [inline index.html]: - [ ] Awesome with the way you layout the form items, but I think we need to follow the requirement that stated "form items should be aligned using flexbox property." 😄

File: index.html (hunk 1 above)
Comment (OyePriscilla) [inline index.html]: - [ ] Kindly remove this comments for cleaner codebase. 👍 Please do the same for line 234 and  and 270 as well. 😄

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @PrinceBadru 😃, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Linters checks passed ✔️ 

## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -37,154 +36,250 @@ <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer
...
+    
+                    </ul>
+    
+                </form>
+    
+                    </div>
+                </section>
 
+
+            </section>
```
Comment (joskalenda) [inline index.html]: - [ ] I think you did not remark that this contact section is duplicated, please remove one and fix the styling before your project gets approved.
- [ ] The image on the side of the get in touch button should stick on the right side, please check the design and see where it is positioned.

duplicate

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @PrinceBadru , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]...✅ Good commit messages
- [ ]...✅ Great use of the inline comment feature
- [ ]...✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @PrinceBadru, @Ferizad and @KALUNGI-J-BOBSON  , 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️
- Linter checks are passing
- Good commit messages
- Required change has been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Yazino12 — COMMENTED** [review]
## Hi @Ferizad @PrinceBadru @KALUNGI-J-BOBSON 🖐🏾,

### Reason for Invalidation:
- In order to submit this project, all of the participants collaborating on this project need to implement the project requirements in all repositories in your pair-programming group [Here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=You%20should%20implement,this%20activity%20individually.). That means that each group member needs to open a new pull request in your repo. **_Your partner cannot submit the project on your behalf!!!_** 👇🏽👇🏽👇🏽

# STATUS: INVALID CODE REVIEW REQUEST 🚫🚫🚫
_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**